openupgrade_scripts/scripts/website_event_questions/14.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,62 @@
+---Models in module 'website_event_questions'---
+obsolete model event.answer (renamed to event.question.answer)
+obsolete model event.question.report [sql_view]
+new model event.question.answer (renamed from event.answer)
+# DONE: pre-migration: renamed tables
+
+---Fields in module 'website_event_questions'---
+website_event_questions / event.event              / question_ids (one2many)       : now a function
+# NOTHING TO DO
+
+website_event_questions / event.question           / answer_ids (one2many)         : relation is now 'event.question.answer' ('event.answer') [nothing to do]
+website_event_questions / event.question           / is_individual (boolean)       : DEL
+website_event_questions / event.question           / once_per_order (boolean)      : NEW
+website_event_questions / event.question           / question_type (selection)     : NEW required, selection_keys: ['simple_choice', 'text_box'], req_default: function, hasdefault
+# NOTHING TO DO
+
+website_event_questions / event.registration       / answer_ids (many2many)        : DEL relation: event.answer
+website_event_questions / event.registration       / registration_answer_ids (one2many): NEW relation: event.registration.answer
+# NOTHING TO DO
+
+website_event_questions / event.registration.answer / event_answer_id (many2one)    : DEL relation: event.answer, required
+website_event_questions / event.registration.answer / event_registration_id (many2one): DEL relation: event.registration, required
+website_event_questions / event.registration.answer / question_id (many2one)        : NEW relation: event.question, required
```

Comment:
This field should be filled in post-migration:

```sql
UPDATE event_registration_answer era
SET question_id = eqa.question_id
FROM event_question_answer eqa
WHERE era.value_answer_id = eqa.id
```

Reply:
@MiquelRForgeFlow I missed this field. Done bro